.github/actions/spelling/allow.txt

```diff
@@ -1,519 +1,522 @@
-makeappx
-MAKEINTRESOURCE
-makemsix
-MANIFESTSCHEMA
-MANIFESTVERSION
-MBs
-mday
-metadata
-microsoft
-mimetype
-Minimatch
-MINORVERSION
-monostate
-motw
-msbuild
-msdata
-MSDN
-msi
-msiexec
-msix
-msixbundle
-msixinfo
-msixsdk
-msixsdkx
-msixtest
-msrc
-Multifile
-Multimatch
-mutex
-namespace
-namespaces
-Nelon
-netcoreapp
-newid
-NOCLOSEPROCESS
-nodiscard
-noexcept
-nomem
-NOMINMAX
-NONAME
-nonexistentsetting
-NONINFRINGEMENT
-NOTHROW
-NOTIMPL
-NOTNULL
-npos
-NTFS
-NTSTATUS
-nuget
-nullptr
-nullsoft
-nupkg
-nuspec
-nunit
-OAuth
-ofstream
-opencode
-opensource
-ostream
-ostringstream
-OSVERSIONINFOEXW
-outfile
-OUTOFMEMORY
-OWC
-PACKAGESSCHEMA
-Params
-params
-parentidx
-pathpart
-Pathto
-PBYTE
-pch
-PCWSTR
-pdb
-PEVENT
-pfp
-PGP
-PII
-pipssource
-placeholders
-png
-posix
-powershell
-PPV
-pplx
-pragma
-PRECONFIG
-preindexed
-prepareforpackaging
-PRIMARYKEY
-prioritization
-PRODUCTNAME
-PRODUCTVERSION
-psz
-ptr
-publiccontainer
-PUCHAR
-PVOID
-pwa
-QCol
-RAII
-rdbuf
-README
-READONLY
-READWRITE
-realloc
-REALTIME
-Redistributable
-regex
-regexp
-removemanifest
-repolibtest
-rescap
-resheader
-resmimetype
-RESTSOURCE
-resw
-resx
-roadmap
-robuffer
-rowcount
-rowid
-rubengustorage
-runsettings
-runtimes
-safecast
-savepoint
-SCROLLER
-SCROLLVIEWER
-sdk
-seekg
-seinfo
-selectany
-SERVICEPACKMAJOR
-SERVICEPACKMINOR
-setfill
-setschemaversion
-setvariable
-setw
-shcore
-shellapi
-SHELLEXEC
-SHELLEXECUTEINFO
-SHELLEXECUTEINFOA
-SHELLEXECUTEINFOW
-Shlwapi
-signtool
-silentwithprogress
-simplesave
-simpletest
-sizeof
-sln
-snprintf
-sourced
-Specv
-sprintf
-sql
-sqlbuilder
-sqlite
-sqliteicu
-sqliteindex
-sqliteindexsource
-sqlitewrapper
-srand
-src
-srwlock
-sscanf
-sstream
-STATFLAG
-STATSTG
-stdcall
-stderr
-stdexcept
-stdout
-STGM
-stmt
-stoi
-stoll
-stoull
-strcoll
-streamoff
-streampos
-stricmp
-strikethrough
-STRINGID
-STRINGIFY
-STRINGIZE
-stringstream
-strstr
-SUBLANG
-subresource
-subselect
-substr
-SWIPECONTROL
-SYMED
-Sys
-sz
-TARG
-TARGETDIR
-targetentrypoint
-targetnametoken
-tdbuild
-tdd
-tellg
-tempdb
-terabyate
-testcontainer
-testmoniker
-Testrun
-testsettingname
-TEXTFORMAT
-TEXTINCLUDE
-there're
-Timeline
-todo
-tolower
-toupper
-towlower
-TRACELOGGING
-triaged
-trunc
-TRUSTEDPEOPLE
-tt
-ttl
-typedef
-typename
-UAC
-uap
-UBool
-UBreak
-ubrk
-uchar
-ucol
-UCollation
-UCollator
-UError
-UIA
-UIF
-uint
-Uknown
-ulong
-ULONGLONG
-uncomment
-und
-undef
-unicode
-uninstall
-uninstalling
-Unregister
-updatemanifest
-UPLEVEL
-upvote
-uregex
-URegular
-uri
-url
-urlmon
-userguide
-usersources
-utext
-utf
-uuidof
-validator
-valijson
-vcxitems
-vcxproj
-vdproj
-versioned
-VERSIONINFO
-versioning
-VERYSILENT
-VFT
-visualstudio
-vm
-VOS
-vso
-wapproj
-wchar
-wcout
-wcsicmp
-webpage
-wekyb
-wil
-WINAPI
-WINEVENT
-winget
-WININET
-winmeta
-winres
-winrt
-winsqlite
-wix
-wmain
-woah
-wofstream
-workflow
-wostream
-wpfn
-wrl
-WStr
-wstring
-wstringstream
-www
-xamarin
-xaml
-xlang
-xml
-xmlns
-XResource
-xsd
-XTOKEN
-yaml
-yml
-yy
+abi

```

Comment:
You would assume something like that, but I haven't spotted it yet.

Reply:
fwiw, you can enable actions in your local fork and push to a branch not tied to this PR to test.

Reply:
```
jsoref@Joshs-MacBook-Pro winget-cli % git show 444f4c4545a7aab99c0c9e5e6db00c0e9cb9fca1  commit 444f4c4545a7aab99c0c9e5e6db00c0e9cb9fca1 (HEAD -> sreadingMSFT/comapireview)
Author: sreadingMSFT <74242768+sreadingMSFT@users.noreply.github.com>
Date:   Tue Apr 27 01:04:44 2021 -0700

    Fix spellcheck

diff --git a/.github/actions/spelling/allow.txt b/.github/actions/spelling/allow.txt
index c69cbae..e520dd3 100644
--- a/.github/actions/spelling/allow.txt
+++ b/.github/actions/spelling/allow.txt
@@ -1,519 +1,522 @@
-abi
-ACCESSDENIED
-addmanifest
...
-yaml
-yml
-yy
+abi^M
+ACCESSDENIED^M
+addmanifest^M
+addstore^M
+admins^M
+Alloc^M
...
```

So, yes, your line endings changed to Windows.

But I thought that check-spelling tolerated that...

Lemme try prerelease...

Reply:
https://github.com/check-spelling/winget-cli/runs/2447125892?check_suite_focus=true

> Warning: .github/actions/spelling/expect.txt: line 1, columns 0-5, Warning - ignoring entry because it contains non alpha characters (non-alpha-in-dictionary)

Is the complaint.

I really thought that I had tolerated Windows line endings.

Well, I guess I'm glad I waited a bit...

Reply:
ok, prerelease now warns and tolerates this instead of complaining and dropping the items:
https://github.com/check-spelling/winget-cli/runs/2447705918?check_suite_focus=true

That leaves me back w/ one thing I want to clean up, and then I should be able to release.

But, in the interim, please run `dos2unix` or click the `CRLF` indicator in the status bar of VSC and select LF.